src/modules/flow/iio/nodes.c

```diff
@@ -106,10 +108,12 @@ create_device_channels(struct gyroscope_data *mdata, int device_id)
 static int
 gyroscope_open(struct sol_flow_node *node, void *data, const struct sol_flow_node_options *options)
 {
-    struct gyroscope_data *mdata = data;
+    struct iiodevice_data *mdata = data;
     const struct sol_flow_node_type_iio_gyroscope_options *opts;
     int device_id;
 
+    SOL_WRN("%s-%s:%s\n", __func__, __DATE__, __TIME__);
```

Comment:
never use `__DATE__` and `__TIME__` in logs. The logs already include the function name and `\n`.